src/components/Navbar/index.css

```diff
@@ -8,23 +8,28 @@ $navbar-shadow: 0 2px 11px rgba(0, 0, 0, 0.1);
 $menu-icon-size: 30px;
 
 .root {
-  display: grid;
-  align-items: center;
+  display: flex;
   justify-content: space-between;
-  grid-template-columns: repeat(2, auto);
-  grid-template-rows: $line-height-body;
-
-  padding-top: $spacing-28;
+  padding: $spacing-12 0;
 
   a {
     text-decoration: none;
   }
 
   @include Breakpoint-mobileOnly {
+    display: grid;
```

Comment:
You don't need grid on mobile navbar, right?

Reply:
I do, for the menu itself.

Reply:
So, we have also change to IE, right?

Reply:
There is no menu in IE, unless the user changes the dimensions of the page

Reply:
Forget 🤦‍♀️